src/components/molecules/LoginForm/LoginForm.styled.ts

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+
+import Typography from '@components/atoms/Typography';
+import { boxShadow } from '@styles/snippets';
+
+export const FormWrapper = styled.div`
+  background-color: #ffffff;
+  border-radius: ${({ theme }) => theme.borderRadius[8]};
+  padding: 2rem;
+  ${boxShadow}
+`;
+
+export const Subheading = styled(Typography)`
+  text-transform: capitalize;
+`;
+
+export const Paragraph = styled.p`
+  text-align: center;
+`;
```

Comment:
Use the Typography component here. You can use the `as` prop to declare it as a `p` element (i.e. `<Typography as='p' />`)
```suggestion
export const Paragraph = styled(Typography)`
  text-align: center;
`;
```